examples/getting_started/README.md

```diff
@@ -35,11 +38,38 @@ A single line of the Redpajama V2 JSONL file has the structure denoted below. Si
 The two raw dataset splits for training and evaluation can be found in 
 `data/raw/redpajama_v2_samples_512_train.jsonl` and `data/raw/redpajama_v2_samples_512_test.jsonl`
 and need to be preprocessed into the [MemMap dataset format](https://github.com/Modalities/modalities/blob/main/src/modalities/dataloader/dataset.py). 
-Firstly, we create the dataset index via
 
+### Config File
+To do so, we employ the `example_dataset_config_train.yaml` and `example_dataset_config_test.yaml` configuration files, which contain the paths of the input and output files, the path of the tokenizer as well as some configurable parameters:
+```
+# example_dataset_config_train.yaml
+
+settings:
+  src_path: data/raw/redpajama_v2_samples_512_train.jsonl
+  dst_path: data/mem_map/redpajama_v2_samples_512_train.pbin
+  index_path: data/mem_map/redpajama_v2_samples_512_train.idx
+  jq_pattern: .raw_content
+  num_cpus: ${node_env:num_cpus}
+  eod_token: <|endoftext|>
+
+tokenizer:
+  component_key: tokenizer
+  variant_key: pretrained_hf_tokenizer
+  config:
+    pretrained_model_name_or_path: tokenizer
+    padding: false
+    max_length: 1024
```

Comment:
for the example, maybe we want to go with  a context length of 512?

Reply:
Yes, I initially chose 1024 here to differentiate this from the number of documents in the example data, which is also 512. But I changed the context length back to 512 now in e391cfc2646dcb52e9943449cc42f665f3b3b5fc.